tools/language-server/wstest/set-expression-value/README.md

```diff
@@ -0,0 +1,62 @@
+# set-expression-value
+
+Benchmark for command that sets the expression value.
+
+## Run
+
+Build Enso distribution.
+
+```bash
+sbt buildEngineDistribution
+```
+
+Start the Language Server and redirect ouput to `language-server.log`
+
+```bash
+built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/enso \
+  --log-level trace \
+  --server \
+  --root-id 6f7d58dd-8ee8-44cf-9ab7-9f0454033641 \
+  --path tools/language-server/wstest/set-expression-value/Unnamed/ \
+  --rpc-port 30616 \
+  --data-port 30717 \
+  &| tee language-server.log
+```
+
+Run the test
+
+```bash
+cargo run -p wstest -- \
```

Comment:
In which directory am I supposed to run the test?
```
enso./tools/language-server/wstest/set-expression-value$ cargo run -p wstest --   --binary-socket ws://127.0.0.1:30717   --init-binary-socket tools/language-server/wstest/set-expression-value/init.bin   --init-text-socket tools/language-server/wstest/set-expression-value/init.txt   --wait-after-init 10000   --ignore-text-socket-responses tools/language-server/wstest/set-expression-value/ignore_responses.txt   --input tools/language-server/wstest/set-expression-value/input.txt   --input-expects-binary-responses   --input-repeat-times 20   ws://127.0.0.1:30616
    Finished dev [unoptimized] target(s) in 0.13s
     Running `/enso/target/rust/debug/wstest --binary-socket 'ws://127.0.0.1:30717' --init-binary-socket tools/language-server/wstest/set-expression-value/init.bin --init-text-socket tools/language-server/wstest/set-expression-value/init.txt --wait-after-init 10000 --ignore-text-socket-responses tools/language-server/wstest/set-expression-value/ignore_responses.txt --input tools/language-server/wstest/set-expression-value/input.txt --input-expects-binary-responses --input-repeat-times 20 'ws://127.0.0.1:30616'`
Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

Reply:
Got the right directory - e.g. root directory of the repository. However the last message is:
```
[info] [2022-07-13T08:12:15.95499808Z] [main] wstest sent bench request [{ "jsonrpc": "2.0", "method": "text/applyExpressionValue", "id": 0, "params": { "expressionId": "524dd815-b652-4bbe-b9f2-26b35d17993a", "expressionValue":"9", "edit": { "path":{"rootId":"6f7d58dd-8ee8-44cf-9ab7-9f0454033641","segments":["src","Main.enso"]}, "edits":[{"range":{"start":{"line":4,"character":15},"end":{"line":4,"character":16}}, "text":"9"}], "oldVersion":"e980e79db6f43cfb85f74b642a5df42b46ff6827ae191e2551d82456", "newVersion":"6dea845964972ebdceccc36e3d4605324fc54bde40709e5d35c1a93e"} } }]
[info] [2022-07-13T08:12:15.958890969Z] [main] wstest received response [{"jsonrpc":"2.0","id":0,"error":{"code":-32601,"message":"Method not found"}}]
```